client/components/Room.jsx

```diff
@@ -3,12 +3,12 @@ Room = React.createClass({
 
   getMeteorData () {
     const messagesHandle = Meteor.subscribe("chatterMessages", {
-      roomId: this.props.room,
+      roomId: this.props.roomId,
```

Comment:
this.props.roomId blir brukt 3 ganger, heller definere en const øverst
`const { roomId } = this.props;`
Og bruke denne?